webapp/ascii_art.py

```diff
@@ -34,5 +53,18 @@ def upload_file():
             file.save(filepath)
             img = Image.open(filepath)
             ascii_ = convert_image_to_ascii(img)
+            # file.save("./gallery") Giving Permission denied error so just used image folder instead. Solve it if you can please
+            file.close()
```

Comment:
I'm guessing that you're getting "permission denied" either because the "gallery" folder doesn't exist, or it *does* exist and therefore you cannot save a file with the exact same name (`file.save` needs a full path, including a file name).
In any case, I don't think we need that line because reading the images from the `IMG_FOLDER` does the job, no need to duplicate the images.

Reply:
> I'm guessing that you're getting "permission denied" either because the "gallery" folder doesn't exist, or it _does_ exist and therefore you cannot save a file with the exact same name (`file.save` needs a full path, including a file name). In any case, I don't think we need that line because reading the images from the `IMG_FOLDER` does the job, no need to duplicate the images.

yes that makes sense. I will remove it in a new commit